### DISCUSSION TITLE
Is it possible to have the app directories in one place and just the index.html in the actual directory

### DISCUSSION THREAD

**Discussion by Andis59** [DirectoryLister/DirectoryLister#911] [Help & Support]
I wonder if it's possible to have the app directories in one place and just distribute the index.html to the directory that contains the files?
It seems a bit of a waste to have the same files all over...

I have a directory(data/files) that has (today) about 60 sub-directories where the files are.
It should not be possible to leave the sub-directory!!

I don't know PHP!

**Comment by PHLAK** [comment]
Unfortunately configuring a different "base" directory is not currently possible. At the moment the Directory Lister code and files to be listed must be in the same folder.

This is due to the way Directory Lister handles (or rather doesn't handle) file access. Directory Lister only lists the files. When it comes to accessing (i.e. downloading) those files it defers to the web server to handle. While Directory Lister could list files outside of the web server's root folder the web server would not be able to access those files for download.

Someday in the future we may add file handling to Directory Lister which would allow this to be implemented but for now it's not in scope.
